accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -510,12 +604,42 @@ default Void run(SystemUnderTest sut) throws Throwable
         }
     }
 
+    public static class SimpleCommand<State> implements UnitCommand<State, Void>
```

Comment:
a few tests don't have an explicit `SystemUnderTest`, so this tends to shorten things